### PR TITLE
accounting for Nonbinary gender on advanced filtering

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
`Nonbinary` will need to be added to the `reporting.gender` table in order for this new value to be viewable

new checkbox input
screen shot 2018-03-23 at 4 15 37 pm

### CODE REVIEW

File: reporting-service/src/main/resources/application-no-permission.sql.yml [hunk 1]
```diff
@@ -75,6 +75,11 @@ sql:
         join ethnicity e on se.ethnicity_id=e.id
       where se.student_id=:student_id
 
+  gender:
+    findAll: >-
+      select id, code
+      from gender
```
Comment (esotrope) [inline reporting-service/src/main/resources/application-no-permission.sql.yml]: - id + explicit order

Comment (dsnyder-fairway) [reply reporting-service/src/main/resources/application-no-permission.sql.yml]: done

**Comment by malaffoon** [comment]
I really don't care which but is it going to be "NonBinary" or "Nonbinary"? I think Devin has it correct as "Nonbinary" but just want to make sure so i'm consistent when i make the back-end changes. @jtreuting ?

**Comment by jtreuting** [comment]
@malaffoon I'm guessing you mean the code that is sent over in the TRT?  for that I don't care either way.  what about making the ingest check case insensitive.

for the label, i believe that should be "Non-Binary".  it is hyphenated when i google it.

**Comment by dsnyder-fairway** [comment]
@jtreuting I went with Nonbinary to follow the comment on the sow

> Per the discussion with CDE yesterday, please replace references to "other" with "nonbinary" (no hyphen). This should appear in the filters with All, Male, and Female.

**Comment by jtreuting** [comment]
okay, leave it for now and i'll double check on that since it doesn't fit with what i'm seeing on google.

**Comment by dsnyder-fairway** [comment]
@jtreuting @malaffoon @esotrope is it okay to merge this?

**Comment by esotrope** [comment]
can you wait till i finish merging the hotfix back into develop then i think you're good to merge

**Pull request merged**